components/site/Footer.tsx

```diff
@@ -235,7 +235,8 @@ export default function Footer() {
           <div>
             <a
               className="text-dark small"
-              href="/zasady-zpracovani-osobnich-udaju"
+              href="/terms
+              "
```

Comment:
I would keep the link as it was, we want to use the czech URL:

```suggestion
              href="/zasady-zpracovani-osobnich-udaju"
```